jcli/src/jcli_lib/transaction/simplified.rs

```diff
@@ -0,0 +1,218 @@
+use crate::jcli_lib::rest::RestArgs;
+use crate::jcli_lib::transaction::{common, Error};
+use crate::transaction::mk_witness::WitnessType;
+use crate::transaction::staging::Staging;
+use crate::utils::key_parser::read_ed25519_secret_key_from_file;
+use crate::utils::AccountId;
+use crate::{rest, transaction};
+use chain_addr::Kind;
+use chain_core::property::FromStr;
+use chain_crypto::{Ed25519, Ed25519Extended, PublicKey, SecretKey};
+use chain_impl_mockchain::account::SpendingCounter;
+use chain_impl_mockchain::key::EitherEd25519SecretKey;
+use chain_impl_mockchain::transaction::Output;
+use jormungandr_lib::interfaces;
+
+use crate::rest::v0::message::post_fragment;
+use crate::transaction::common::CommonFees;
+use crate::utils::io::ask_yes_or_no;
+use jormungandr_lib::interfaces::SettingsDto;
+use rand::rngs::OsRng;
+use rand::SeedableRng;
+use rand_chacha::ChaChaRng;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct MakeTransaction {
+    /// the account to debit the funds from
+    #[structopt(name = "ACCOUNT")]
+    pub sender_account: interfaces::Address,
+
+    /// the value
+    #[structopt(name = "VALUE")]
+    pub value: interfaces::Value,
+
+    /// the account to debit the funds from
+    #[structopt(long)]
+    pub receiver: Option<interfaces::Address>,
+
+    #[structopt(long)]
+    pub block0_hash: String,
+
+    /// the file path to the file to read the signing key from.
+    /// If omitted it will be read from the standard input.
+    #[structopt(long)]
+    pub secret: Option<PathBuf>,
+
+    /// Set the change in the given address
+    #[structopt(long)]
+    pub change: Option<interfaces::Address>,
+
+    #[structopt(flatten)]
+    pub common: common::CommonTransaction,
+
+    #[structopt(flatten)]
+    rest_args: RestArgs,
+
+    // force transaction without requesting for confirmation
+    #[structopt(long)]
+    force: bool,
+
+    #[structopt(long)]
+    post: bool,
+}
+
+impl MakeTransaction {
+    pub fn exec(self) -> Result<(), Error> {
+        let secret_key = read_ed25519_secret_key_from_file(&self.secret)?;
+        let receiver_address = if let Some(address) = self.receiver {
+            address
+        } else {
+            let (_, address) = create_receiver_secret_key_and_address()?;
+            address
+        };
+        let transaction = make_transaction(
+            self.sender_account,
+            receiver_address,
+            secret_key,
+            self.value,
+            &self.block0_hash,
+            self.rest_args.clone(),
+            self.change,
+            self.force,
+        )?;
+
+        transaction.store(&self.common.staging_file)?;
+
+        if self.post {
+            let fragment = transaction.fragment()?;
+            let fragment_id = post_fragment(self.rest_args, fragment)?;
+            println!("{}", fragment_id);
```

Comment:
Same about adding more information here: right now we are just printing numbers and letters.

Reply:
Likewise here, but it is clearer here that we could output something better, since this is more a one-shot interactive thingy :)